--- a/SConstruct	2018-04-15 21:43:38.000000000 +0200
+++ b/SConstruct	2018-09-09 18:02:43.128113454 +0200
@@ -3341,7 +3341,7 @@
 				return 'brcmEGL'
 			return self.default_EGL_LIB
 		def __default_DATA_DIR(self):
-			return '%s/share/games/%s' % (self.prefix, self._program.target)
+			return '%s/share/games/%s' % (self.prefix, self._program.shortname)
 		def _generic_variable(key,help,default):
 			return (key, help, default)
 		def __get_configure_tests(tests,_filter=lambda s: s.name[0] != '_'):
