--- a/rust/Cargo.toml	2017-08-02 14:39:01.000000000 +0200
+++ b/rust/Cargo.toml	2017-09-09 01:56:04.717760707 +0200
@@ -4,27 +4,13 @@
 authors = ["Stefan Lau <github@stefanlau.com>"]
 release-debuginfo = true
 
-[profile.dev]
-rpath = true
-[profile.release]
-rpath = true
-
 [lib]
 crate-type = ["dylib"]
 name = "stracciatella"
 path = "src/stracciatella.rs"
 
 [dependencies]
-getopts = { git = "https://github.com/rust-lang-nursery/getopts", rev = "b8beaf63" }
+getopts = "0.2"
 libc = "0.2"
 serde = "0.8"
 serde_json = "0.8"
-
-[dev-dependencies]
-regex = "0.1"
-tempdir = "0.3"
-
-[target.'cfg(windows)'.dependencies]
-winapi = "0.2"
-user32-sys = "0.2"
-shell32-sys = "0.1"
