--- a/opt/brother/Printers/BrGenML1/cupswrapper/brother_lpdwrapper_BrGenML1	2014-05-14 06:36:11.000000000 +0200
+++ b/opt/brother/Printers/BrGenML1/cupswrapper/brother_lpdwrapper_BrGenML1	2016-10-21 15:06:30.099225191 +0200
@@ -50,7 +50,9 @@
 }
 chomp($basedir);
 
-$basedir =~ s/$PRINTER\/cupswrapper\/.*$/$PRINTER\//g;
+#$basedir =~ s/$PRINTER\/cupswrapper\/.*$/$PRINTER\//g;
+my $PRINTER = "BrGenML1";
+my $basedir = "/opt/brother/Printers/$PRINTER";
 
 
 %units = ( pt   => \&pt2dot, 
@@ -77,16 +79,16 @@
 
 my $cmdoptions=$ARGV[4];
 my $PPD = $ENV{PPD};
-my $PRINTER=$basedir;
+#my $PRINTER=$basedir;
 my $CUPSINPUT='';
 if ( @ARGV >= 6 ){
     $CUPSINPUT=$ARGV[7];
 }
 
 
-$PRINTER =~ s/^\/opt\/.*\/Printers\///g;
-$PRINTER =~ s/\/cupswrapper//g;
-$PRINTER =~ s/\///g;
+#$PRINTER =~ s/^\/opt\/.*\/Printers\///g;
+#$PRINTER =~ s/\/cupswrapper//g;
+#$PRINTER =~ s/\///g;
 
 for  (my $i = 0 ; $i < @ARGV ; $i ++){
     logprint( 0 , "ARG$i      = $ARGV[$i]\n");
@@ -733,7 +735,7 @@
     my $lpddir = $basedir;
     my %lpr_options = %$lpr_options_ref;
 
-    $lpddir = $basedir."/lpd/";
+    $lpddir = $basedir."/lpd";
     my $lpdconf = $lpddir.'/'.$LPDCONFIGEXE;
 
     
